lib/participant.js

```diff
@@ -454,6 +462,18 @@ class Participant extends EventEmitter {
  * @event Participant#networkQualityLevelChanged
  */
 
+/**
+ * The {@link Participant} has reconnected to the {@link Room} after a signaling connection disruption.
+ * @param {Participant} participant - The {@link Participant} that has reconnected.
+ * @event Participant#reconnected
+ */
+
+/**
+ * The {@link Participant} is reconnecting to the {@link Room} after a signaling connection disruption.
+ * @param {Participant} participant - The {@link Participant} that is reconnecting.
+ * @event Participant#reconnecting
```

Comment:
Are these 'reconnecting' 'reconnected' events applicable for LocalParticipant as well? 
Isn't local participant reconnecting is same as room reconnecting/ reconnected ?

Reply:
@makarandp0 Yes, they do apply to LocalParticipant and they are only fired if the reconnection is due to Signaling interruption.